Assets/SEE/Layout/NodeLayouts/IncrementalTreeMap/CorrectArea.cs

```diff
@@ -1,46 +1,55 @@
 using System.Linq;
 using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using MathNet.Numerics.LinearAlgebra;
 using SEE.Game.City;
 
 namespace SEE.Layout.NodeLayouts.IncrementalTreeMap
 {
     /// <summary>
     /// Provides an algorithm to recalibrate the layout, so that the areas of the <see cref="Rectangle"/>
-    /// of <see cref="Node"/> match the wanted <see cref="Node.Size"/> of the node.
+    /// of <see cref="Node"/> match the <see cref="Node.DesiredSize"/> of the node.
     /// </summary>
     internal static class CorrectAreas
     {
         /// <summary>
         /// Recalibrate the layout, so that the areas of the <see cref="Rectangle"/>
-        /// of <see cref="Node"/> match the wanted <see cref="Node.Size"/> of the node.
+        /// of <see cref="Node"/> match the <see cref="Node.DesiredSize"/> of the node.
         /// </summary>
         /// <param name="nodes">nodes with layout</param>
-        /// <param name="settings">the settings</param>
-        /// <returns></returns>
+        /// <param name="settings">the settings of the incremental tree map layout</param>
+        /// <returns>true if correction was successful, else false</returns>
         public static bool Correct(IList<Node> nodes, IncrementalTreeMapSetting settings)
         {
-            if (nodes.Count == 1) return true;
+            if (nodes.Count == 1)
+            {
+                return true;
+            }
+
             if (IsSliceAble(nodes, out Segment slicingSegment))
             {
                 Split(nodes, slicingSegment,
                     out IList<Node> partition1,
                     out IList<Node> partition2);
 
+                // adjust the position of slicingSegments
                 AdjustSliced(nodes, partition1, partition2, slicingSegment);
+                // recursive adjust the two sub layouts
+                // since both sublayouts are temporally independent from each other
+                // the segment that separates these must be considered as border (IsConst = true)
```

Comment:
```suggestion
                // recursively adjust the two sub layouts
                // since both sublayouts are temporally independent from each other
                // the segment that separates these must be considered as a border (IsConst = true)
```